index.html

```diff
@@ -42,12 +44,12 @@ <h1>Compose cloud infrastructure and services into custom platform APIs</h1>
           alt="Provision and manage cloud infrastructure" />
       </div>
       <div class="col-6_md-12">
-        <h2>Provision and manage cloud infrastructure and services using kubectl</h2>
+        <h2>Don’t start from scratch</h2>
         <p>
-          Crossplane extends your Kubernetes cluster, providing you with CRDs for any infrastructure
-          or managed service. Compose these granular resources into higher level abstractions that
-          can be versioned, managed, deployed and consumed using your favorite tools and existing
-          processes you've already integrated with your clusters.
+          The best platforms are built on control planes, but historically
+          building a control plane has required writing code—a lot of code. With
+          Crossplane you can build a control plane tailored to your unique needs
+          in days, not months, without writing tricky distributed systems code.
```

Comment:
"complicated" instead of "tricky"? not a strong feeling on this

Reply:
Maybe. I was going more for "hard to get right", which I think "tricky" captures better despite feeling a little informal.